transport/ws/src/main/java/org/kaazing/gateway/transport/ws/bridge/extensions/idletimeout/IdleTimeoutExtensionFactory.java

```diff
@@ -31,6 +31,12 @@ public String getExtensionName() {
     }
 
     @Override
+    public WebSocketExtension offer(ExtensionHelper extensionHelper, WsResourceAddress address) {
+        // connectors are not sending this extension in handshake request
```

Comment:
I see 

In WSN Connector
```
        // (KG-7391) Use ping and pong to detect and close dead connections, if ws inactivity timeout is active
        final ResourceAddress connectAddress = (ResourceAddress) session.removeAttribute(WSN_CONNECT_ADDRESS_KEY);
        WsCheckAliveFilter.addIfFeatureEnabled(filterChain, WsnAcceptor.CHECK_ALIVE_FILTER,
                connectAddress.getOption(WsResourceAddress.INACTIVITY_TIMEOUT), logger);
```

I would expect it would be better to use the extension for this purpose instead of doing it this way.  The difference in the implementations would be that the extension would hit the wire in the extension header, and then the value could be negotiated.  Since it apparently has never worked that way, its probably ok to check it in as it is now and then open an issue to fix it.